src/idsk/components/intro-block/intro-block.scss

```diff
@@ -249,6 +247,33 @@
         padding-right: govuk-spacing(1);
     }
 
+    .app-pane-grey .idsk-intro-block-side-menu-circle svg circle {
+        fill: govuk-colour("black");
+    }
+
+    .app-pane-grey {
+        h2.govuk-heading-l,
```

Comment:
ak je tu class-a netreba davat tag, tak isto namiesto tagov by som daval radsej class-i.. obcas to aj mne ujde no snazme sa tomu vyvarovat je to best practice